api/internal/target/kusttarget_test.go

```diff
@@ -139,10 +139,7 @@ commonLabels:
 
 func TestMakeCustomizedResMap(t *testing.T) {
 	th := kusttest_test.MakeHarness(t)
-	th.WriteK("/whatever", `
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
```

Comment:
remove  `apiVersion: kustomize.config.k8s.io/v1beta1` and `kind: Kustomization`,
because  [th.WriteK](https://github.com/kubernetes-sigs/kustomize/blob/e3981daf0ebca762fe1a29dfd39b70418c6697ba/api/testutils/kusttest/harness.go#L46) is always added above two lines.